tasks/package-apt.yml

```diff
@@ -6,6 +6,7 @@
     state: "{{ item.state | default(package_state) }}"
     update_cache: "{{ package_update_cache }}"
     cache_valid_time: "{{ package_cache_valid_time }}"
+    install_recommends: "{{ item.apt_install_recommends | default(false) }}"
```

Comment:
As the module doesn't provide a default value, you should change this to use the `default(omit)` filter.